src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,98 @@
+package parser;
+
+import com.github.javafaker.Faker;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import shop.Cart;
+import shop.RealItem;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JsonParserTest {
+
+    public static final String RESOURCE_PATH = "src/main/resources/%s.json";
+    private JsonParser jsonParser;
+    private static String cartName;
+    private static String filePath;
+    private static Faker faker;
+    private static ArrayList<String> listOfFiles;
```

Comment:
Use interface as type when possible, it helps to avoid strange not-flexible methods (code) like:
public void doSomething(ArrayList param);
public void doSomething(HashMap param);

In this case type of variable will be List or Collection